src/mol-plugin-state/transforms/model.ts

```diff
@@ -375,6 +400,49 @@ const TrajectoryFromXYZ = PluginStateTransform.BuiltIn({
     }
 });
 
+type TrajectoryFromLammpsData = typeof TrajectoryFromLammpsData
+const TrajectoryFromLammpsData = PluginStateTransform.BuiltIn({
+    name: 'trajectory-from-lammps-data',
+    display: { name: 'Parse Lammps Data', description: 'Parse Lammps Data string and create trajectory.' },
+    from: [SO.Data.String],
+    to: SO.Molecule.Trajectory,
+    params: {
+        unitsStyle: PD.Select('real', PD.arrayToOptions(['real', 'metal', 'si', 'cgs', 'electron', 'micro', 'nano', 'lj'])),
```

Comment:
Should the unit kind/factor be passed here?

Reply:
If we want this params to appears in the gui ( not necessary ), how does that work ?